types/penner/index.d.ts

```diff
@@ -0,0 +1,61 @@
+// Type definitions for penner 0.1
+// Project: https://github.com/bcherny/penner
+// Definitions by: Sergey Koshechkin <https://github.com/FrankiePo>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.8
+
+export = penner;
```

Comment:
does it require UMD support? so `export as namespace`
https://github.com/bcherny/penner/blob/master/penner.js#L38
https://github.com/bcherny/penner/blob/master/penner.coffee#L28

Reply:
https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#identifying-a-umd-library